app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -349,6 +349,12 @@ def parse_container_definition(container_def, pod_id)
       new_result[:container_env_vars] = Array(env).collect do |env_var|
         parse_container_env_var(env_var)
       end
+      fragments = container_def.command
+      new_result[:container_command_fragments] = Array(fragments).collect do |fragment|
```

Comment:
@blomquisg instead of having an entirely new table only for commands, would it be an acceptable solution to serialize it to json in the database? (I don't expect doing selection over this column)